lib/rubocop/cop/style/safe_navigation.rb

```diff
@@ -40,6 +40,12 @@ module Style
       #   foo.nil? || foo.bar
       #   !foo || foo.bar
       #
+      #   # Methods that are used on assignment, arithementic operation or
+      #   comparison should not be converted to use safe navigation
```

Comment:
You forgot an `#` on the second line of the comment. You also forgot to regenerate the manual.